### DISCUSSION TITLE
Using 'gdf_to_nx()',  I am not sure if I'm passing the additional weight data correctly

### DISCUSSION THREAD

**Discussion by seangrogan** [pysal/momepy#505] [Q&A] [answered]
Hello there.  

I have a GeoDataFrame that has a network of trails, using momepy.gdf_to_nx() I was attempting to make a graph and get a path using nx.astar_path()

The distance seems to be accurate, but I want to minimize time on the path, not distance.  The gdf has a column "TRAVEL_TIME" that I'd like to use instead of a distance metric, however it appears that  "TRAVEL_TIME" isn't being passed to the Graph; when running `nx.astar_path(G, Start, End, weight="TRAVEL_TIME")` or `nx.astar_path_length(G, Start, End, weight="TRAVEL_TIME")` I get the number of edges in the AStar path, not the sum of the time.  It doesn't appear the additional information is being passed to the graphs

I am not sure if I should pass the information differently or if there is something else I'm missing.

Thanks

**Comment by martinfleis** [comment]
Would you be able to provide a small reproducible example? The columns should be available on the graph normally but it is possible that you are hitting some unknown bug.

**Comment by seangrogan** [comment, marked as answer]
I had a data problem, not a python problem

So when creating the example, it didn't work, but I typo'd the column "TRAVEL_TIME"; fixing it, the results are as expected.

Upon some further exploration, the shapefile column name "TRAVEL_TIME" was actually TRAVELTIME and the readme was wrong. Feel a little silly, thanks for your response.  

Best,
